demos/TicTacToe/Classes/main_menu_scene.cc

```diff
@@ -670,8 +688,8 @@ void MainMenuScene::update(float /*delta*/) {
         }
       }
     } else if (current_state_ == kWaitingSignUpState) {
-      if (user_result_.status() == firebase::kFutureStatusComplete) {
-        if (user_result_.error() == firebase::auth::kAuthErrorNone) {
+      if (user_result_.status() == kFutureStatusComplete) {
```

Comment:
since there aren't any else statements here, can this and the other if statements like it be :

```
if (user_result_.status() == kFutureStatusComplete && user_result_.error() == kAuthErrorNone ) {
```